components/13-ironic/aio-values.yaml

```diff
@@ -47,12 +47,42 @@ conf:
       automated_clean: false
     dhcp:
       dhcp_provider: none
+    oslo_messaging_rabbit:
+      rabbit_ha_queues: true
   logging:
     logger_root:
       level: DEBUG
     logger_ironic:
       level: DEBUG
 
+endpoints:
+  oslo_messaging:
+    namespace: null
+    auth:
+      admin:
+        username: rabbitmq
+        password: password
+        secret:
+          tls:
+            internal: rabbitmq-tls-direct
+      ironic:
+        username: ironic
+        password: password
+    statefulset:
+      replicas: 3
+      name: rabbitmq-server
+    hosts:
+      default: rabbitmq-nodes
+    host_fqdn_override:
+      default: null
```

Comment:
This isn't necessary cause this is the default when you set `statefulset` and `hosts` like you did above.